docs/source/5-administrator.md

```diff
@@ -234,6 +237,14 @@ FlowMachine usage log:
 
 Note that the `request_id` field is identical across the five log entries, which lets you match the request across the multiple services.
 
+### Error and Debugging Logs
+
+By default, FlowMachine and FlowAPI write error logs to stdout. For more verbose logging, set the `LOG_LEVEL` environment variable to `debug` when starting the docker container.
+
+FlowMachine also supports writing debug and error logs to a file. To enable this, set the `WRITE_LOG_FILE` environment variable to `TRUE` when starting the FlowMachine container. Logs will be written to `$LOG_DIRECTORY/flowmachine-debug.log` (by default `LOG_DIRECTORY=/var/log/flowmachine`). If the directory specified does not exist, FlowMachine will attempt to create it.
+
+Log messages from FlowMachine will show the `logger` field of the log entry as the Python module that emitted the log entry (e.g. `{'logger':'flowmachine.core.query'..}`. FlowAPI debugging messages set `logger` to `flowapi`.
```

Comment:
For consistency with flowapi, replace `logger` with `flowmachine` in the flowmachine logs?

Reply:
I think the use case is a little different though - in flowmachine, the logger tells you which class wrote the message, useful for debugging. That's not really relevant in flowapi, cos the debug logger is a global app wide one.

Reply:
Oh sorry, ignore me. I misunderstood what this sentence was trying to explain...

Do we need to update the "Complete Logging Cycle" example then? I'm not sure the `logger` fields in this example represent what is being explained here?

Reply:
Personally, I don't think so. In 'normal' operation, you should never see debugging messages, and they aren't relevant to following the execution for audit purposes.